2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,10 +19,15 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+  let newObject={};
+  for (elem of countryCurrencyCodes){
```

Comment:
What's missing from this line? Can you think of a better variable name than `elem`?

Reply:
Yes "let" is missing and we can use countryCurrencyCode instead of elem